TRANSLATION.md

```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
+
+Hozirgi jarayon: 
+https://github.com/reactjs/uz.reactjs.org/issues/1
+
+## Tarjima bo'yicha kelishuv
+
+| Terminning asli | Tarjim |
+| ------------------ | ---------- |
+| API reference | API ma'lumotnoma |
+| array | massiv |
+| arrow function |  arrow funksiya, strelka funksiya  |
+| attribute | atribut |
+| batch | yangilash guruhi |
+| batching | guruhlash |
+| bug | bag, xatolik  |
```

Comment:
What about `kamchilik` ?

Reply:
hmm, maybe, but i think it not compatible variant.
Examples: 
- "there's a bug -> Kodingda kamchilik bor". 
- "It is a bug -> "bu kamchilik"